src/fromager/commands/download_sequence.py

```diff
@@ -55,9 +55,12 @@ def download_sequence(
                 )
 
             if include_wheels:
-                sdist.download_wheel(
-                    wkctx,
-                    req,
-                    wkctx.wheels_downloads,
-                    wheel_servers,
-                )
+                try:
+                    sdist.download_wheel(
+                        wkctx,
+                        req,
+                        wkctx.wheels_downloads,
+                        wheel_servers,
+                    )
+                except Exception as err:
+                    logger.error(f"Failed to download {req}: {err}")
```

Comment:
```suggestion
                    logger.error(f"Failed to download wheel for {req}: {err}")
```
just a nitpick to differentiate whether the failure was in downloading sources or wheels